Shippo/Track.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Shippo {
+    [JsonObject (MemberSerialization.OptIn)]
+    public class Track : ShippoId {
+        [JsonProperty (PropertyName = "carrier")]
+        public object Carrier { get; set; }
```

Comment:
We need the getters for it to be initially filled out, else they will always be NULL